crates/rust-analyzer/src/config.rs

```diff
@@ -1743,6 +1756,22 @@ fn field_props(field: &str, ty: &str, doc: &[&str], default: &str) -> serde_json
                 { "type": "null" }
             ],
         },
+        "CargoUnsetTest" => set! {
+            "type": ["string", "array"],
+            "items": { "type": "string" },
+            "enum": ["all"],
+            "enumDescriptions": [
+                "All crates are analyzed without `--cfg=test`",
+            ],
+        },
```

Comment:
This needs to use `anyOf` now for validity reasons in vscode, check the current version of `"CargoFeatures"` of what I mean

Reply:
Done.